web/src/compose_send_menu_popover.ts

```diff
@@ -79,12 +82,12 @@ export function open_schedule_message_menu(
                 );
             }
             const $popper = $(instance.popper);
-            const message_schedule_callback = (time) => {
+            const message_schedule_callback = (time: string | number): void => {
```

Comment:
Is this really inconsistent in type? Seems like it's likely a bug elsewhere if that's the case.

Reply:
I think `time` might always be a string here?
```
export function show_flatpickr(                                                                         
    element: HTMLElement,                                                                               
    callback: (time: string) => void,                                                                   
```

Reply:
Yeah I'm trying to focus on just typing things correctly, but we could open an issue to fix it if you think it's worth looking into soon.

Reply:
In general, I think of type-checking as a means towards good code, so when we find bad code using it, we should fix it. But I'm not convinced this is particulary bad, so we can let it be.